app/src/main/java/com/hedvig/app/feature/referrals/ui/tab/ReferralsAdapter.kt

```diff
@@ -462,14 +471,35 @@ class ReferralsAdapter(
                 data: ReferralsModel,
                 reload: () -> Unit,
                 tracker: ReferralsTracker,
-                marketManager: MarketManager
+                marketManager: MarketManager,
+                onBannerClicked: () -> Unit
             ) {
                 binding.retry.setHapticClickListener {
                     tracker.reload()
                     reload()
                 }
             }
         }
+
+        class InfoBanner(parent: ViewGroup) : ViewHolder(parent.inflate(R.layout.view_info_banner)) {
```

Comment:
Should we implement this using Compose instead?

Reply:
Yes :D